monai/apps/pathology/transforms.py

```diff
@@ -0,0 +1,182 @@
+# modified from sources:
+# - Original implementation from Macenko paper in Matlab: https://github.com/mitkovetta/staining-normalization
+# - Implementation in Python: https://github.com/schaugf/HEnorm_python
+# - Link to Macenko et al., 2009 paper: http://wwwx.cs.unc.edu/~mn/sites/default/files/macenko2009.pdf
+import cupy as cp
```

Comment:
Please make this an optional import, so that if it does not exist, it doesn't affect other components. Like this:
https://github.com/Project-MONAI/MONAI/blob/99bca69f26f9f6a76e60f2ce7848e83470547833/monai/apps/pathology/handlers.py#L22-L25